omero/util_scripts/Dataset_To_Plate.py

```diff
@@ -51,22 +51,23 @@ def add_image_to_plate(conn, image, plate_id, column, row, remove_from=None):
     well.plate = omero.model.PlateI(plate_id, False)
     well.column = rint(column)
     well.row = rint(row)
-    well = update_service.saveAndReturnObject(well)
 
     try:
-        ws = omero.model.WellSampleI()
-        ws.image = omero.model.ImageI(image.id, False)
-        ws.well = well
-        well.addWellSample(ws)
-        update_service.saveObject(ws)
+        for image in images:
+            ws = omero.model.WellSampleI()
+            ws.image = omero.model.ImageI(image.id, False)
+            ws.well = well
+            well.addWellSample(ws)
+        update_service.saveObject(well)
```

Comment:
Mostly out of curiosity: why did you change from ws to well?

Reply:
I use ```ws``` for each WellSample, but now we have many new ```ws``` and I want to save them all in one shot, I can do that by saving the ```well```.